.storybook/config.js

```diff
@@ -8,6 +11,29 @@ import 'core-js/fn/object/create';
 import 'core-js/fn/object/define-properties';
 import 'core-js/fn/object/set-prototype-of';
 
+// Option defaults.
+addDecorator(
+  withOptions({
+    name: 'AXA Living Styleguide',
+    theme: {
+      ...themes.normal,
+      mainTextFace: '"Source Sans Pro", Arial, sans-serif;',
+      menuLink: {
+        color: 'black',
+      },
+      brand: {
+        height: '55px',
+        paddingLeft: '65px',
+        background: 'url("https://axa.ch/etc/clientlibs/axa/frontend-lib/images/logo-AXA.svg") no-repeat',
```

Comment:
I wouldn’t do this. Rather local assets folder

Reply:
Fine done.